causal_testing/testing/causal_test_outcome.py

```diff
@@ -57,7 +58,7 @@ def apply(self, res: CausalTestResult) -> bool:
             ci_low = res.ci_low() if isinstance(res.ci_low(), Iterable) else [res.ci_low()]
             ci_high = res.ci_high() if isinstance(res.ci_high(), Iterable) else [res.ci_high()]
             value = res.test_value.value if isinstance(res.ci_high(), Iterable) else [res.test_value.value]
-
+            value = value[0] if isinstance(value[0], pd.Series) else value
```

Comment:
Please can you explain why this was necessary and nothing for the confidence intervals? I think it should have worked as it was before. If not, I'm surprised the confidence intervals didn't break anything

Reply:
After looking at the test files, I also think this should probably work in some sort of loop too, rather than just grabbing the first value